tests/unit/backend/api_server/base_api_test.py

```diff
@@ -87,6 +87,14 @@ def get_cxguser_token(user="owner"):
 class BaseAuthAPITest(BaseAPITest):
     def setUp(self):
         super().setUp()
+
+        # TODO: this can be improved, but the current authorization method requires it
```

Comment:
@Bento007 do you know what's going on here? Looks like the mock below isn't enough since this line:
```x = assert_authorized_token(token, CorporaAuthConfig().curation_audience)```
is calling the config in the parameters, which happens before the mock is called.

Reply:
this is how I'm doing it https://github.com/chanzuckerberg/single-cell-data-portal/blob/9e0a2cb7ecc9b009da4567670fc118cc4355fd7c/tests/unit/backend/api_server/base_api_test.py#L90-L94
I'm letting the config value resolve to the default value and mock the whole `assert_asuthorized_token` function